src/VirtualClient/VirtualClient.Contracts.UnitTests/Parser/LscpuParserTests.cs

```diff
@@ -28,6 +28,18 @@ public void LscpuParserParsesTheExpectedResultsFromIntelSystems_Scenario1()
             Assert.AreEqual(1, info.SocketCount);
             Assert.AreEqual(1, info.NumaNodeCount);
             Assert.IsTrue(info.IsHyperthreadingEnabled);
+            Assert.AreEqual(0, info.MaxFrequencyMHz);
+            Assert.AreEqual(0, info.MinFrequencyMHz);
```

Comment:
0 is potentially dangerous and confusing as actual 0, or someone are doing comparison

Reply:
Agree. I was planning for -1 as default value. I will recheck if and update it.